### ISSUE TITLE
Adjusting Discon File

### ISSUE THREAD

**Issue by mwh50** [OpenFAST/openfast#139]
Hey, two questions:

1.  I'm trying to adjust the pitch control for the 5_MW_Land_DLL_WTurb.  I've been trying to go about that by modifying the Discon solution file in `C:\Users\mhampton\regtest\openfast\vs-build`, and building the changed version.  Unfortunately, this doesn't seem to be doing anything.  The data that comes out when I run regression tests still seems to be the same despite the changed code.  What should I be modifying to implement my own control system?

2.  I'm trying to limit the number of tests I run when I do the regression testing.  I really only care about the 5_MW_... ones, so the others are sort of just taking up time when I run them.  Is there a good way to modify manualRegressionTest.py so that it only runs the tests I want it to?  Ideally, I'd just have multiple modified 5_MW_Land_DLL_WTurb tests run so I could compare different versions to each other.

As always, thanks for helping!  I appreciate any advice you give me!

--Mike

**Comment by rafmudaf** [comment]
```
mbp@~/Development/openfast/reg_tests$ python manualRegressionTest.py -h
usage: manualRegressionTest.py [-h] [-p [Plotting-Flag]] [-n [No-Execution]]
                               [-v [Verbose-Flag]] [-case [Case-Name]]
                               OpenFAST System-Name Compiler-Id Test-Tolerance

Executes OpenFAST and a regression test for a single test case.

positional arguments:
  OpenFAST              path to the OpenFAST executable
  System-Name           current system's name: [Darwin,Linux,Windows]
  Compiler-Id           compiler's id: [Intel,GNU]
  Test-Tolerance        tolerance defining pass or failure in the regression
                        test

optional arguments:
  -h, --help            show this help message and exit
  -p [Plotting-Flag], -plot [Plotting-Flag]
                        bool to include matplotlib plots in failed cases
  -n [No-Execution], -no-exec [No-Execution]
                        bool to prevent execution of the test cases
  -v [Verbose-Flag], -verbose [Verbose-Flag]
                        bool to include verbose system output
  -case [Case-Name]     single case name to execute
```

The manual regression test script does not currently support running multiple cases, but you can use the `-case` flag and write your own script around it to run only the 5MW cases. Or you could modify the existing script to take a variable number of inputs after the `-case` flag and add them to `caselist` at line 63:

```
if case is not "":
    caselist = [case]
```

**Comment by rafmudaf** [comment]
Regarding modifying the DISCON controller, are you copying your newly compiled `DISCON.dll` into the 5MW_Baseline directory before rerunning the test? See this part of the documentation:
https://openfast.readthedocs.io/en/dev/source/testing/regression_test.html#manual-driver-configuration

**Comment by rafmudaf** [comment]
@mwh50 How is this going for you? Did this info help?

**Comment by mwh50** [comment]
@rafmudaf Yes this helped a lot.  I went into the `manualRegressionTest.py` and saw it referenced the `CaseList.md` file.  I then commented out the tests I wasn't interested in running.  Also, I just implemented my new DISCON file using your advice and it worked great!

One final question though.  Has the simulink interface been fixed for OpenFAST?  I was reading about it in [this open issue thread](https://github.com/OpenFAST/openfast/issues/59), but I couldn't tell if it had been fixed in the time since October 2017.  I tried running it earlier today, but wasn't able to get past the issue initially brought up.

Thank you very much!
